Samples/GDS/Client/Controls/ApplicationCertificateControl.cs

```diff
@@ -316,41 +313,48 @@ private async void CertificateRequestTimer_Tick(object sender, EventArgs e)
 
                     if (!String.IsNullOrEmpty(m_application.CertificateStorePath) && !String.IsNullOrEmpty(m_application.CertificateSubjectName))
                     {
-                        CertificateIdentifier cid = new CertificateIdentifier()
-                        {
+                        CertificateIdentifier cid = new CertificateIdentifier() {
                             StorePath = m_application.CertificateStorePath,
                             StoreType = CertificateStoreIdentifier.DetermineStoreType(m_application.CertificateStorePath),
                             SubjectName = m_application.CertificateSubjectName.Replace("localhost", Utils.GetHostName())
                         };
 
                         // update store
-                        using (var store = CertificateStoreIdentifier.OpenStore(m_application.CertificateStorePath))
+                        ICertificateStore store;
+
```

Comment:
TODO: support NoPrivateKey flag in CertStoreIdentifier

Reply:
done with preview, waiting for final release